sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -98,6 +98,7 @@ class RadarLocationReceiver : BroadcastReceiver() {
 
     @SuppressLint("MissingPermission")
     override fun onReceive(context: Context, intent: Intent) {
+        Radar.logger.d("Broadcast Received: ${intent.action}")
```

Comment:
This log line should follow the same format as the others - "Receiving broadcast | ${intent.action}"

Reply:
@jsani-radar it looks like this syntax is commonly used in the SDK _after_ something has occurred. For example, take a look [here](https://github.com/radarlabs/radar-sdk-android/blob/develop/sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt#L93) and the next ten lines. This shows the "-ing" _before_ an event occurs and "-ed" _after_ it comes back. I will fix the parameter here (should say " | intent = ...")